scripts/CheckSecurityConfigs.s.sol

```diff
@@ -111,11 +134,16 @@ contract CheckSecurityConfigs is Script {
         checkAddressIsExpected(addresses.L1CrossDomainMessengerProxy, addresses.L1StandardBridgeProxy, "messenger()");
     }
 
-    function checkL2OutputOracleProxy(ProtocolAddresses memory addresses, bool isMainnet) internal {
+    function checkL2OutputOracleProxy(ProtocolAddresses memory addresses, bool upgradedToFPAC) internal {
+        if (upgradedToFPAC) {
+            // This check is skipped for chains which upgraded to FPAC
+            console2.log("Skipping L2OutputOracleProxy check");
```

Comment:
Is it worthwhile to put the reason we're skipping in here?

Reply:
Done in [e50257f](https://github.com/ethereum-optimism/superchain-registry/pull/139/commits/e50257fd3e30d602cef266c0bd40ea19d57b5540)